app/Http/Controllers/Auth/AuthController.php

```diff
@@ -3,12 +3,125 @@
 namespace App\Http\Controllers\Auth;
 
 use App\Http\Controllers\Controller;
+use App\Models\User;
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Facades\Validator;
 
 class AuthController extends Controller
 {
-    public function login(Request $request)
+    /**
+     * Show login form
+     *
+     * @param Request $request
+     * @return \Illuminate\Contracts\Foundation\Application|\Illuminate\Contracts\View\Factory|\Illuminate\Contracts\View\View
+     */
+    public function login_form()
+    {
+        return view('auth.user.login');
+    }
+
+    /**
+     * Process login form
+     *
+     * @param Request $request
+     * @return \Illuminate\Contracts\Foundation\Application|\Illuminate\Http\RedirectResponse|\Illuminate\Routing\Redirector
+     */
+    public function process_login(Request $request)
+    {
+        $request->validate([
+            'email' => 'required|email',
+            'password' => 'required|min:6',
+        ]);
+
+        $user_data = array(
+            'email' => $request->get("email"),
+            'password' => $request->get("password"),
+        );
+
+        if(Auth::attempt($user_data))
+        {
+            return redirect('/home');
+        }else{
+            return back()->with("error", lang_path("en.auth.failed"));
+        }
+    }
+
+    /**
+     * Show signup form
+     *
+     * @param Request $request
+     * @return \Illuminate\Contracts\Foundation\Application|\Illuminate\Contracts\View\Factory|\Illuminate\Contracts\View\View
+     */
+    public function signup_form()
+    {
+        return view('auth.user.signup');
+    }
+
+    /**
+     * Process signup form
+     *
+     * @param Request $request
+     * @return \Illuminate\Contracts\Foundation\Application|\Illuminate\Contracts\View\Factory|\Illuminate\Contracts\View\View
+     */
+    protected function process_signup(Request $request)
+    {
+        $rules = [
+            'name' => ['required', 'string', 'max:255'],
+            'email' => ['required', 'string', 'email', 'max:255', 'unique:users'],
+            'password' => ['required', 'string', 'min:6', 'confirmed'],
+        ];
+        $validator = Validator::make($request->all(), $rules);
+        if ($validator->fails()) {
+            return redirect('signup')
+            ->withInput()
+            ->withErrors($validator);
+        } else {
+            $data = $request->all();
```

Comment:
if ($validator->fails()) {
            return redirect('signup')
            ->withInput()
            ->withErrors($validator);
        } else {
            $data = $request->all();
            $data['password'] = Hash::make($data['password']);
            try {
                $this->userRepository->create($data);
                return redirect('login')->with('status',lang_path("en.auth.signup_success"));
            } catch (\Exception $exception) {
                return redirect('login')->with('status',lang_path("en.auth.signup_failed"));
            }
        }